pulumi/__main__.py

```diff
@@ -376,7 +376,7 @@ def main():
 
     rsw_db = rds.Instance(
         "rsw-db",
-        instance_class="db.t3.micro",
```

Comment:
Why do we change from db.t3.micro to db.t4g.micro ? Just curious - cost reasons ?